src/pages/StudentRegistration/index.tsx

```diff
@@ -0,0 +1,629 @@
+import { useState } from 'react';
+
+import {
+  Box,
+  Typography,
+  TextField,
+  Button,
+  MenuItem,
+  Grid,
+} from '@mui/material';
+import { useNavigate } from 'react-router';
+
+const genderOptions = ['Masculino', 'Feminino', 'Outro'];
+const raceOptions = [
+  'Branca',
+  'Preta',
+  'Parda',
+  'Amarela',
+  'Indígena',
+  'Outro',
+];
+const escolaridadeOptions = ['Fundamental', 'Médio', 'Superior', 'Outro'];
+const identityTypesOptions = ['RG', 'CPF', 'Certidão de Nascimento'];
+const documentTypesOptions = ['Comprovante de Residência', 'Comprovante de Renda', 'Outro'];
+
+export default function StudentRegistration() {
+  const currentYear = new Date().getFullYear();
+  const years = Array.from({ length: currentYear - 1980 + 1 }, (_, i) => currentYear - i);
+
+  const [personal, setPersonal] = useState({
+    nome: '',
+    nascimento: '',
+    sexo: '',
+    raca: '',
+    escolaridade: '',
+    escolaAtual: '',
+    endereco: '',
+    cep: '',
+    numero: '',
+    complemento: '',
+    ano: '',
+    documentos: '',
+    identidade: '',
+    numeroIdentidade: ''
+  });
+
+  type Documento = {
+    arquivo: string;
+    tipo: string; // extensão
+    tipoDocumento: string; // tipo selecionado pelo usuário
+    origem: string;
+    data: string;
+    descricao: string;
+    id: number;
+  };
+
+  const [documentos, setDocumentos] = useState<Documento[]>([]);
+  const [showUploader, setShowUploader] = useState(false);
+  const [docForm, setDocForm] = useState({
+    arquivo: '',
+    tipo: '',
+    origem: '',
+    data: '',
+    descricao: '',
+  });
+  const [detalhes, setDetalhes] = useState({
+    dataCadastro: '',
+    comoConheceu: '',
+    vinculo: '',
+  });
+  const navigate = useNavigate();
+
+  const handleAddDoc = () => {
```

Comment:
Just to keep current standard (no arrow functions)
```suggestion
  function handleAddDoc() {
```